scripts/tests/misc.sh

```diff
@@ -18,7 +18,7 @@ test_package_with_feature primitives std $rustflags
 
 no_runtime_benchmarks=('court' 'market-commons' 'rikiddo')
 
-cargo test --package zeitgeist-runtime --lib -- --nocapture
+cargo +nightly-2022-09-24 test --package zeitgeist-runtime --lib -- --nocapture
```

Comment:
```suggestion
cargo test --package zeitgeist-runtime --lib -- --nocapture
```

Apply this when https://github.com/zeitgeistpm/zeitgeist/pull/993 is merged.